common/src/test/java/brocoeur/example/common/roulette/strategy/direct/RouletteRiskyStrategyTest.java

```diff
@@ -17,7 +17,7 @@ class RouletteRiskyStrategyTest {
     @Test
     void shouldGetGreenWithRiskyRouletteStrategy() {
         // Given - When
-        var actualPlay = rouletteRiskyStrategy.getStrategyPlay();
+        var actualPlay = rouletteRiskyStrategy.getStrategy();
```

Comment:
Wdyt about val from lombok

Reply:
To see in another "improvement" card cause it impacts all the tests